packages/web-twig/src/Resources/components/Accordion/README.md

```diff
@@ -219,5 +219,21 @@ further extend component's descriptiveness and accessibility. Also, UNSAFE styli
 see the [Escape hatches][escape-hatches] section in README to learn how and when to use them.
 These attributes will be passed to the topmost HTML element of the component.
 
+## JavaScript Plugin
+
+For full functionality, you need to provide Spirit JavaScript:
```

Comment:
Do we really need to copy this text everywhere? This is the same as in the main web package readme. Can we just point to this readme and the proper section?

Reply:
I think this is the MVP we all agreed on, we can further improve it in a next step.

Reply:
Well, ok, but still I think this will be doubled work. 🤷